cadctl/cmd/investigate/investigate.go

```diff
@@ -138,20 +138,13 @@ func run(_ *cobra.Command, _ []string) error {
 		return ccam.Evaluate(cluster, err, ocmClient, pdClient, alertTypeString)
 	}
 
-	logging.Info("Successfully jumpRoled into the customer account. Removing existing 'Cloud Credentials Are Missing' limited support reasons.")
```

Comment:
i think we need to explicitly think about ccam as well. We mainly came from a perspective of removing LS automation for chgm. If we keep posting LS on ccam, my feeling is we need at least some possibility of removing it automatically.